### PR TITLE
Update Downloading-files.md

### PR DESCRIPTION
**Pull request by akaegi**
Hi, I tried to adapt the sample to the newest ASP.NET Core version...
Maybe you can check?
I added the next() "call". I'm not entirely sure if it's correct - I don't know much about the Pipeline but it seems suitable to me.

### CODE REVIEW

**Comment by smatsson** [comment]
Hi,

Thanks for contributing. Looks good with some notes below! 👍 

1. Not really sure why you added a praying emoj to the start of the help file. I should not be there :) 
2. We use the path `/files` for all our examples (and not `/api/files` as in your example).
3. The next call would make calls to `/files/` (instead of `/files/X`) go to the next handler which is probably a good idea. You might also want to add an else statement to your outermost if (`if (context.Request.Path.StartsWithSe...`) to run the next handler if the url does not match `/files` at all.

**Comment by akaegi** [comment]
Thanks for the review. I adjusted these three points.
Furthermore I added a _using_ statement around the stream returned from `GetContentAsync`

File: Downloading-files.md [hunk 1]
```diff
@@ -4,41 +4,53 @@ The following example requires that the data store implements `ITusReadableStore
...
+                }
+                else
+                {
+                        // Call next handler in pipeline if it's something else
+                        await next();
+                }
+        }
+        else
+        {
+                await _next();
```
Comment (smatsson) [inline Downloading-files.md]:
```suggestion
                await next();
```

**Comment by smatsson** [comment]
Thanks for the PR! Merged.

**Pull request merged**